### PR TITLE
feat: 🎸 add two new cards to whatsnew

### PR DESCRIPTION
**Pull request by paulstey**
BREAKING CHANGE: 🧨 no

This PR just refreshes the `What's New` section with a couple of new cards. 

#### Check the types of changes present in this PR:
- [ ] :bug: Bug
- [ ] :dragon: Feature
- [ ] :frog: Data (data folder - people/opportunities)
- [x] :dog: Content (content folder)
- [ ] :blowfish: Other. Specify:

#### Checklist:
- [x] Commitizen was used for all commits.
- [x] Local site looks as expected after changes.
- [x] Contributing guidelines were followed.
- [x] If changes affect development, was the README updated?

##### If PR to `production`
- [ ] Development site (https://datasci.brown.edu) was checked and looks as expected.

### CODE REVIEW

**Review by singhsaluja — APPROVED** [review]

File: data/whatsnew.yaml [hunk 1]
```diff
@@ -6,13 +6,13 @@ items:
...
+  - title: myBrown Research Tab
+    text: There is a new Research Compute tab on the myBrown portal! [**Check out the myBrown Portal!**](https://my.brown.edu/web/guest/research-compute)
     fa:
       prefix: fad
-      icon: fa-boot
-  - title: Oscar "How-To" Videos
-    text: CCV User Services has updated the Oscar documentation to include brief "how-to" videos for new Oscar users. [**Check out vidoes.**](https://docs.ccv.brown.edu/oscar/short-videos)
+      icon: fa-microscope
+  - title: Python Workshop
+    text: CCV will be hosting an introductory Python workshop. [**Check out Python workshop details.**](https://today.brown.edu/events/139280?utm_source=todayAtBrown&utm_medium=email&utm_campaign=Campus%20Faculty)
```
Comment (mcmcgrath13) [inline data/whatsnew.yaml]: we probably don't want those utm sources in the url, right?

File: data/whatsnew.yaml [hunk 2]
```diff
@@ -6,13 +6,13 @@ items:
     fa:
       prefix: fad
       icon: fa-door-open
-  - title: Bootcamp
-    text: CCV will be hosting a virtual bootcamp for HPC as well as computational biology! [**Check out the bootcamp details!**](https://docs.ccv.brown.edu/bootcamp2021)
+  - title: myBrown Research Tab
+    text: There is a new Research Compute tab on the myBrown portal! [**Check out the myBrown Portal!**](https://my.brown.edu/web/guest/research-compute)
```
Comment (mcmcgrath13) [inline data/whatsnew.yaml:10]: does everyone have the research compute tab? if not, what happens if they follow this link and they don't?

**Review by mcmcgrath13 — CHANGES_REQUESTED** [review]

File: data/whatsnew.yaml (hunk 2 above)
Comment (paulstey) [reply data/whatsnew.yaml:10]: Thanks for checking, @mcmcgrath13! And yes, everyone should be able to see that tab. Do you not see it?

File: data/whatsnew.yaml (hunk 1 above)
Comment (paulstey) [reply data/whatsnew.yaml]: Fixed! Thank you, @mcmcgrath13!!

File: data/whatsnew.yaml (hunk 2 above)
Comment (mcmcgrath13) [reply data/whatsnew.yaml:10]: I did! But I didn't know if all would (undergrads, e.g.)

**Review by mcmcgrath13 — APPROVED** [review]

**Comment by paulstey** [comment]
Thank you, @mcmcgrath13 and @singhsaluja!!!

**Pull request merged**